integrations/lib/embeddedtbot/bot.go

```diff
@@ -29,16 +29,16 @@ import (
 
 	"github.com/gravitational/teleport/api/client"
 	"github.com/gravitational/teleport/api/client/proto"
-	"github.com/gravitational/teleport/lib/tbot"
+	"github.com/gravitational/teleport/lib/tbot/bot"
+	"github.com/gravitational/teleport/lib/tbot/bot/connection"
 	"github.com/gravitational/teleport/lib/tbot/bot/destination"
-	"github.com/gravitational/teleport/lib/tbot/config"
 	"github.com/gravitational/teleport/lib/tbot/services/clientcredentials"
 )
 
 // EmbeddedBot is an embedded tBot instance to renew the operator certificates.
 type EmbeddedBot struct {
 	log *slog.Logger
-	cfg *config.BotConfig
+	cfg bot.Config
```

Comment:
Just to be sure, `bot.Config` config doesn't contain locks anymore? IIRC I had to make this a pointer to avoid copying a lock.